client/src/views/PatientList.vue

```diff
@@ -419,7 +418,9 @@ export default Vue.extend({
           formValues.patientStatus = undefined
         }
 
-        formValues.quarantineStatus = this.getQuarantineSelection() as Array<PatientStatus>
+        formValues.quarantineStatus = this.getQuarantineSelection() as Array<
+          PatientStatus
+        >
```

Comment:
Komisch. Bei mir wird das immer genau anders herum formatiert. Muss mal in meine settings schauen

Reply:
Bei mir liefen nur die Tools aus dem Buildprozess die mir das hier ausgespuckt haben. An der Stelle keine IDE-Intervention.